src/Sulu/Bundle/ReferenceBundle/Domain/Model/ReferenceInterface.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ReferenceBundle\Domain\Model;
+
+interface ReferenceInterface
+{
+    public const RESOURCE_KEY = 'references';
+    public const LIST_KEY = 'references';
+
+    public function getId(): int;
+
+    public function getResourceKey(): string;
+
+    public function setResourceKey(string $resourceKey): static;
+
+    public function getResourceId(): string;
+
+    public function setResourceId(string $resourceId): static;
+
+    public function getReferenceLocale(): string;
+
+    public function setReferenceLocale(string $referenceLocale): static;
+
+    /**
+     * @return array<string, string>
+     */
+    public function getReferenceViewAttributes(): array;
```

Comment:
maybe we should name this `referenceRouterAttributes` as currently I think RouterAttributes was used for this in PHP: https://github.com/sulu/sulu/blob/d0fa61f761ee08bfc36bba4bc14b86e305155fc3/src/Sulu/Bundle/AdminBundle/Admin/View/ListViewBuilder.php#L237

In frontend currently it is just called `attributes` on the Router.js.

@Prokyonn what do you think?

Reply:
When we rename this to `referenceRouterAttributes`, should we then also rename the `resource_view_attributes_property` in the [DetailLinkItemAction](https://github.com/sulu/sulu/blob/2.6/src/Sulu/Bundle/AdminBundle/Resources/js/views/List/itemActions/DetailLinkItemAction.js#L16C13-L16C46) to `resource_route_attributes_property`? 🤔